### PR TITLE
Adding asset type

### PR DESCRIPTION
**Pull request by adamdavies1**
Adding asset type column and test date for use in assessment query

### CODE REVIEW

File: db/models/assessment_record/enums.py [hunk 1]
```diff
@@ -17,3 +17,21 @@ class Language(Enum):
 
     en = 0
     cy = 1
+
+
+class AssestType(Enum):
```
Comment (gidsg) [inline db/models/assessment_record/enums.py:22]: Typo `AssestType`

File: db/models/assessment_record/enums.py [hunk 2]
```diff
@@ -17,3 +17,21 @@ class Language(Enum):
...
+    """Status The ENUM used by `db.models.AssessmentRecord` to validate the
+    possible values for the `language` column."""
```
Comment (gidsg) [inline db/models/assessment_record/enums.py:24]: typo `language`

File: db/models/assessment_record/assessment_records.py [hunk 3]
```diff
@@ -49,6 +50,10 @@ class AssessmentRecord(db.Model):
         "workflow_status", ENUM(Status), index=True, default="NOT_STARTED"
     )
 
+    asset_type = db.Column(
+        "asset_type", db.String(255), index=True, nullable=False
```
Comment (gidsg) [inline db/models/assessment_record/assessment_records.py:54]: Why is this a String here but modelled as an enum?

Comment (adamdavies1) [reply db/models/assessment_record/assessment_records.py:54]: @gidsg Sorry, this was a draft PR just to help us look at the changes. Ive just removed the enum stuff now

File: db/models/assessment_record/enums.py (hunk 1 above)
Comment (adamdavies1) [reply db/models/assessment_record/enums.py:22]: This has been removed

File: db/models/assessment_record/enums.py (hunk 2 above)
Comment (adamdavies1) [reply db/models/assessment_record/enums.py:24]: This has been removed

**Review by gidsg — APPROVED** [review]
LGTM 👍

**Pull request merged**